architectures

```diff
@@ -2,3 +2,4 @@ bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
 s390x default,onbuild,slim,stretch
+arm64 default,onbuild,slim,stretch
```

Comment:
That's correct, this cannot have Alpine yet, since the Alpine official image is not officially multiarch yet.